ansible/group_vars/all.yml

```diff
@@ -4,8 +4,12 @@
 #
 # localBuild - requires make release to have been run to build local binaries
 # packageManager - will install packages from your distribution using yum/dnf/apt
+# ghReleases - will download binaries from github releases
 source_type: localBuild
 
+# This version will be used only when installing kubernetes from github releases
+kubernetes_version: v1.0.4
```

Comment:
Found some solution for get latest release:
* Remove `kubernetes_version` from defaults
* Add task:
```
- name: Determine latest release
  shell: /bin/curl https://github.com/kubernetes/kubernetes/releases/latest 2>/dev/null | /bin/perl -n -e'/(https.*)(v\d([^\"]*))/ && print $2'
  when: kubernetes_version is undefined
```
* Comment out `kubernetes_version` in group_vars for those, who wants some exact release.

***

About a task:
```
curl https://github.com/kubernetes/kubernetes/releases/latest
```
will get something like 
```
<html><body>You are being <a href="https://github.com/kubernetes/kubernetes/releases/tag/v1.0.4">redirected</a>.body></html>
```
with is redirect to latest release.
And
```
/bin/perl -n -e'/(https.*)(v\d([^\"]*))/ && print $2'
```
just regex to extract version from curl job.
I would prefer to use Python for this (since it is guarantee be in ansible managed nodes), but not familiar with it =/